### DISCUSSION TITLE
snapshot browser not working

### DISCUSSION THREAD

**Discussion by Simox-wolf** [garethgeorge/backrest#418] [Q&A]
I have install Backrest on my server, and I really liked the program, fast, simple, efficient.
after a couple of days I'm trying the restore but I see that it doesn't let me browse the snapshots.
The server are install on Windows and backup other windows server folder, i have upgrade today to 1.4 release
![image](https://github.com/user-attachments/assets/6322c14d-6aaa-495e-ba88-e81aea9ed559)

can someone help me?

**Comment by garethgeorge** [comment]
Are you still running into an error / problems browsing here?

1.4 isn't expected to have changed anything w.r.t. browsing handling. Do you have any logs of errors you might be running into? For the snapshot browser, any API errors will show up in chrome developer tools e.g. under the JavaScript console.

As a side note -- can you check that other commands work on your repo? E.g. using the `Run Command` feature does the `ls` command work?
